### PR TITLE
Make home page prettier

### PR DESCRIPTION
**Pull request by chance-cyphers**

### CODE REVIEW

File: src/view.ts [hunk 1]
```diff
@@ -115,29 +115,44 @@ export const divider = (): DividerBlock => {
...
+        divider(),
+
+        markdownSection("Here are your open action items:"),
+        ...actionItemsList(items)
     ]
-};
\ No newline at end of file
+};
+
+const actionItemsList = (items: ActionItem[]) => {
```
Comment (chance-cyphers) [inline src/view.ts:136]: I KNOW there's a cleaner and less mutable way to do this stuff...

Comment (chance-cyphers) [reply src/view.ts:136]: Oh and also properly leveraging typescript

File: src/handlers/event.handler.ts [hunk 2]
```diff
@@ -45,8 +45,7 @@ const updateHomeTab = async (userId: string, workspaceId: string, client: Client
     const averageTimeString = moment.duration(avgMs).humanize();
 
     const itemDescriptions = result.rows
-        .filter(row => row.status === 'OPEN')
-        .map(row => row.description);
+        .filter(row => row.status === 'OPEN');
 
     const homeViewBlocks = homeView(averageTimeString, completeds.length, itemDescriptions);
```
Comment (tchupp) [inline src/handlers/event.handler.ts:51]: `itemDescriptions` needs renaming now, right?

File: src/view.ts [hunk 3]
```diff
@@ -115,29 +115,44 @@ export const divider = (): DividerBlock => {
     }
 };
 
-export const homeView = (avg: string, completedCount: number, items: string[]) => {
+const formLink = "https://docs.google.com/forms/d/e/1FAIpQLSe1SltHxH47haVZzKe1x6eLsC89WmdEWtOTr_jo1sxg9t-jQw/viewform?usp=sf_link";
+
+export const homeView = (avg: string, completedCount: number, items: any[]) => {
```
Comment (tchupp) [inline src/view.ts:123]: Is there a stronger type we could apply to `items: any[]`?

Comment (chance-cyphers) [reply src/view.ts:123]: whoops. Created it and didn't apply it here. Will do

File: src/handlers/event.handler.ts (hunk 2 above)
Comment (chance-cyphers) [reply src/handlers/event.handler.ts]: good catch

**Pull request merged**